Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -10,9 +10,12 @@
 using System.Collections.Generic;
 using Snowflake.Data.Log;
 using System.Collections.Specialized;
+using System.Linq;
+using System.Reflection;
 using System.Web;
 using System.Security.Authentication;
 using System.Runtime.InteropServices;
+using System.Runtime.Versioning;

```

Comment:
Please remove the using added as they are not actually being used.

Reply:
Removed